frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -33,9 +33,8 @@ function getSentryLinks(user: UserType | null): string {
         return ''
     }
     const cloud = window.location.origin == 'https://eu.posthog.com' ? 'EU' : 'US'
```

Comment:
We actually have `preflight?.region` that you should use for this instead.

Reply:
and it always works even when logged out?